pkg/operator/controller/machineconfig/controller.go

```diff
@@ -77,11 +77,11 @@ func New(mgr ctrl.Manager, impls ...impl) (*MachineConfigReconciler, error) {
 // and labels the nodes where the changes are to be applied.
 func (r *MachineConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ctrl.Result, err error) {
 
-	if r.Log, err = logr.FromContext(ctx); err != nil {
-		return
+	if ctxLog, err := logr.FromContext(ctx); err == nil {
+		r.Log = ctxLog
 	}
 
-	r.Log.V(3).Info("Reconciling NodeObservabilityMachineConfig of Nodeobservability operator")
+	r.Log.V(1).Info("Reconciling NodeObservabilityMachineConfig of Nodeobservability operator")
```

Comment:
I was trying to follow community guide https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use

Level one is defined as:
* Information about config (listening on X, watching Y)
* Errors that repeat frequently that relate to conditions that can be corrected (pod detected as unhealthy)

This log (and most of others in this file) do not meet the v=1 criteria